buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginBuildPlugin.groovy

```diff
@@ -169,21 +165,42 @@ public class PluginBuildPlugin extends BuildPlugin {
         project.artifacts.add('zip', bundle)
     }
 
+    /** Find the reponame. */
+    static String urlFromOrigin(String origin) {
```

Comment:
Is this still used?

Reply:
yes, by a build script at [distribution/archives/build.gradle](https://github.com/elastic/elasticsearch/blob/master/distribution/archives/build.gradle) we could probably have that use nebula and clean it up, but didn't want to get into that now.